core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -374,6 +374,8 @@ object SparkSubmit {
       OptionAssigner(args.jars, YARN, CLUSTER, clOption = "--addJars"),
 
       // Other options
+      OptionAssigner(args.executorCores, STANDALONE, ALL_DEPLOY_MODES,
+        sysProp = "spark.deploy.maxCoresPerExecutor"),
```

Comment:
please revert this change. I don't think we want to confuse `--executor-cores` with the new config